Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN \
 # ✄---------------------------------------------------------------------
 # This is an actual target container:
 
-FROM python:3.7-alpine
+FROM python:alpine3.8
```

Comment:
```suggestion
FROM python:3.7-alpine3.8
```

Reply:
you need to swith it to tags tab. Official images has vulnerabilties scanner from Docker.

https://hub.docker.com/_/python?tab=tags&page=1

Reply:
Ok, seems like that's only available when having and being logged into a Dockerhub account.

Anyways: Which vulnerabilities that would affect molecule are in Alpine 3.9 but not 3.8?